src/grammar/grammar.ohm

```diff
@@ -93,6 +98,10 @@ Tact {
              | external "(" Parameter? ")" "{" Statement* "}" --externalRegular
              | external "(" stringLiteral ")" "{" Statement* "}" --externalComment
 
+    AsmInstruction = id
+                   | integerLiteralDec
+                   | integerLiteralHex
```

Comment:
Asm instructions can also have various symbols in them, including, but not limited to: `"`, `:`, etc.
For simplicity from the perspective of Tact, I suggest considering them as anything, but spaces or Tact comments.

So, the suggestion is:
```suggestion
    asmInstruction = ~comment (~space any)+
```

Notice the lowercase name to prevent from optional spaces occuring in-between things. The `AsmFunction` rule on line 25 also has to use the lowercase version then :)

Reply:
@novusnota Nice catch! Please checkout my update. I had to introduce explicit grammar category for data, otherwise there is a conflict with the curly braces from the enclosing block